ui.apps/src/main/content/jcr_root/apps/asset-share-commons/clientlibs/clientlib-theme/semanticui-dark/themes/dark/views/card.variables

```diff
@@ -3,7 +3,7 @@
 *******************************/
 
 /* Card */
-@width: 255px;
+@width: 303px;
```

Comment:
@godanny86 iirc you tweaked this value a bit; thoughts on if this will be OK? I assume its to prevent line-wrap when "Remove from Cart" text is present (which makes sense).
ill install locally to test, just a preemptive Q :)